kili/graphql_client.py

```diff
@@ -83,7 +84,8 @@ def _send(self, query, variables):
 
         if self.session is not None:
             try:
-                number_of_trials = 10
+                number_of_trials_env = os.getenv("KILI_SDK_TRIALS_NUMBER")
+                number_of_trials = number_of_trials_env if number_of_trials_env is not None else 10
```

Comment:
```suggestion
                number_of_trials = os.getenv("KILI_SDK_TRIALS_NUMBER", 10)
```

Reply:
I need to manage if the env variable value is not a string, I pushed new changes